_data/team.yml

```diff
@@ -84,3 +84,14 @@
       - programming
       - video games
       - guitar
+      
+- name: Chloe Herman
```

Comment:
Awesome! Can you move this up between my entry, and Chris' entry, that way its sorted by alphabetical order of last names (excluding @gregcaporaso).

Reply:
@thermokarst, Done. Thank you!